cmd/stack_validate.go

```diff
@@ -92,7 +92,7 @@ Runs the following validation tests against the stack and its templates:
 
 			// lint
 			if !noLint {
-				_, err = RunAppsodyCmdExec([]string{"stack", "lint"}, stackPath)
+				_, err = RunAppsodyCmdExec([]string{"stack", "lint"}, stackPath, rootConfig.Verbose)
```

Comment:
Wondering if we should pass just `rootConfig` and let the function extract `Verbose` - rather than explicitly passing in a boolean. This has been a common practice throughout Appsody CLI. Unless there are strong objections, I'd request this change.